.github/workflows/api.yml

```diff
@@ -0,0 +1,33 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [ "main" ]
+    paths: [
+      'api/**',
+      '.github/workflows/api.yml'
+    ]
```

Comment:
Can we keep it to always run? I'd rather optimize this only if build times cause issues. I'm sure we will quickly forget that this needs to be updated in case we introduce some shared code.

Reply:
Sure. Removed.